src/serializers/type_serializers/timedelta.rs

```diff
@@ -26,7 +33,28 @@ impl BuildSerializer for TimeDeltaSerializer {
         _definitions: &mut DefinitionsBuilder<CombinedSerializer>,
     ) -> PyResult<CombinedSerializer> {
         let timedelta_mode = TimedeltaMode::from_config(config)?;
-        Ok(Self { timedelta_mode }.into())
+        let temporal_mode = TemporalMode::from_config(config)?;
+
+        let prefer_timedelta_mode = config
```

Comment:
Again, not sure on my implementation here, its all i could really think of. Happy to take feedback